Okt/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:20.04 AS build
+
+ENV GO_VERSION=1.19.2
+ENV ROCKSDB_VERSION=v6.27.3
+ENV GO_PATH=/go
+ENV OKT_LATEST_VERSION=https://api.github.com/repos/okx/exchain/releases/latest
+ENV DEBIAN_FRONTEND=noninteractive
+
+WORKDIR /home
+
+# At the current time of writing this Dockerfile, there are no nodes available on newer versions upward of 1.6.7.7, so we revert to that version.
+ENV BEFORE_HARD_FORK=true
```

Comment:
Change this variable to version. And if we do not provide specific version, build the latest, otherwise build the provided one.